test/WebJobs.Script.Tests.Integration/SamplesEndToEndRestartTests.cs

```diff
@@ -1,79 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Net.Http;
-using System.Threading.Tasks;
-using Newtonsoft.Json.Linq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    // Running these in their own class with private fixtures so they do not affect the timing of
-    // subsequent tests.
-    public class SamplesEndToEndRestartTests
-    {
-        [Fact(Skip = "Will not pass until https://github.com/Azure/azure-webjobs-sdk-script/issues/1537 is fixed")]
```

Comment:
My concern is that your unit test isn't exercising the problem that found the bug (accessing a disposed ScriptHost). I could be reading it wrong, but it doesn't look like the test creates a ScriptHost, so it will never try to use it.

If that's true, then I'm going to renable LoggerFactory disposal and now I'll need to manually verify it because we have no test that exercises the original issue. I'm 99% sure that this will fix it, but I'd hate to not actually test the original problem.